presidio-image-redactor/Pipfile

```diff
@@ -10,6 +10,7 @@ presidio-analyzer = ">=2.2.0"
 pillow = ">=9.0"
 pydicom = ">=2.3.0"
 pypng = ">=0.20220715.0"
+python-gdcm = ">=3.0.22"
```

Comment:
@niwilso Sharon added python-gdcm to the main branch. On your PR, please just add the NOTICE and remove the pipfile.lock.